mines/beta[]  beta
string frame_id
